syntax = "proto2";
package o_data;

message vector {
  required int32 t = 1;
  repeated double vec_value = 2;
}

message matrix {
  required int32 iter = 1;
  repeated vector vec = 2;
}

message full_o_it {
  repeated matrix m = 1;
}